lib/controllers/ephemeral/account_dashboard_controller.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:get/get.dart';
+import 'package:pispapp/controllers/app/account_controller.dart';
+import 'package:pispapp/models/account.dart';
+import 'package:pispapp/models/transaction.dart';
+import 'package:pispapp/repositories/interfaces/i_transaction_repository.dart';
+import 'package:pispapp/utils/log_printer.dart';
+
+class AccountDashboardController extends GetxController {
+  AccountDashboardController(this._transactionRepo);
+  Account selectedAccount;
+  List<Transaction> transactionList = <Transaction>[];
+
+  ITransactionRepository _transactionRepo;
+
+  void refreshAll() {}
```

Comment:
intended to be empty? or perhaps unimplemented? If unimplemented perhaps can add a todo comment

Reply:
Implemented in the PR following this.

Reply:
okay, gotcha, let's keep this unresolved for future reference

Reply:
addressed in https://github.com/mojaloop/pisp-demo-app-flutter/pull/12